crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -93,3 +93,48 @@ fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
 
     return output_color;
 }
+
+struct Vertex {
+    @location(0) position: vec3<f32>,
+    @location(1) normal: vec3<f32>,
+    @location(2) uv: vec2<f32>,
+#ifdef VERTEX_TANGENTS
+    @location(3) tangent: vec4<f32>,
+#endif
+};
+
+struct VertexOutput {
+    @builtin(position) clip_position: vec4<f32>,
+    @location(0) world_position: vec4<f32>,
+    @location(1) world_normal: vec3<f32>,
+    @location(2) uv: vec2<f32>,
+#ifdef VERTEX_TANGENTS
+    @location(3) world_tangent: vec4<f32>,
+#endif
+};
+
+@vertex
+fn vertex(vertex: Vertex) -> VertexOutput {
```

Comment:
Does this actually work? The standard material only loads the fragment shader from this file, but uses the default vertex shader which is defined in mesh.wgsl.

Also, I think the vertex stuff should go before the fragment shader since that's how all other shaders in bevy are defined.